tests/contrib/pylons/test_pylons.py

```diff
@@ -35,160 +37,142 @@ def start_response(self, status, headers):
         self.out_headers = headers
 
     def start_response_exception(self, status, headers):
-        e = Exception("Some exception")
+        e = Exception('Some exception')
         e.code = 'wrong formatted code'
         raise e
 
     def start_response_string_code(self, status, headers):
-        e = Exception("Custom exception")
+        e = Exception('Custom exception')
         e.code = '512'
         raise e
 
     def start_response_exception_code_method(self, status, headers):
         raise ExceptionWithCodeMethod('Exception with code method')
 
 
-def test_pylons():
-    writer = DummyWriter()
-    tracer = Tracer()
-    tracer.writer = writer
-    app = FakeWSGIApp()
-    traced = PylonsTraceMiddleware(app, tracer, service="p")
-
-    # successful request
-    eq_(writer.pop(), [])
-    app.code = '200 OK'
-    app.body = ['woo']
-    app.environ = {
-        'REQUEST_METHOD':'GET',
-        'pylons.routes_dict' : {
-            'controller' : 'foo',
-            'action' : 'bar',
+class PylonsTestCase(TestCase):
```

Comment:
Re-organized tests in a class to make it reusable later. No changes have been made.